### ISSUE TITLE
Error after upgrade

### ISSUE THREAD

**Issue by ihsanudin** [deployphp/deployer#436]
I have this error after upgrade deployer to version 3.0.8 but in version 3.0.6 all function is running well

```
➤ Executing task deploy:prepare
⤷ on [prod]
PHP Notice:  Cannot connect to vinotiliving.com:313. Error 110. Connection timed out in phar:///usr/local/bin/deployer/vendor/phpseclib/phpseclib/phpseclib/Net/SSH2.php on line 1011
PHP Stack trace:
PHP   1. {main}() /usr/local/bin/deployer:0
PHP   2. require() /usr/local/bin/deployer:4
PHP   3. Deployer\Deployer->run() phar:///usr/local/bin/deployer/bin/dep:62
PHP   4. Symfony\Component\Console\Application->run() phar:///usr/local/bin/deployer/src/Deployer.php:99
PHP   5. Symfony\Component\Console\Application->doRun() phar:///usr/local/bin/deployer/vendor/symfony/console/Application.php:126
PHP   6. Symfony\Component\Console\Application->doRunCommand() phar:///usr/local/bin/deployer/vendor/symfony/console/Application.php:195
PHP   7. Symfony\Component\Console\Command\Command->run() phar:///usr/local/bin/deployer/vendor/symfony/console/Application.php:878
PHP   8. Deployer\Console\TaskCommand->execute() phar:///usr/local/bin/deployer/vendor/symfony/console/Command/Command.php:259
PHP   9. Deployer\Executor\SeriesExecutor->run() phar:///usr/local/bin/deployer/src/Console/TaskCommand.php:82
PHP  10. Deployer\Task\Task->run() phar:///usr/local/bin/deployer/src/Executor/SeriesExecutor.php:39
PHP  11. call_user_func() phar:///usr/local/bin/deployer/src/Task/Task.php:73
PHP  12. {closure:phar:///usr/local/bin/deployer/recipe/common.php:67-93}() phar:///usr/local/bin/deployer/src/Task/Task.php:73
PHP  13. Deployer\Server\Remote\PhpSecLib->connect() phar:///usr/local/bin/deployer/recipe/common.php:68
PHP  14. phpseclib\Net\SFTP->login() phar:///usr/local/bin/deployer/src/Server/Remote/PhpSecLib.php:54
PHP  15. call_user_func_array() phar:///usr/local/bin/deployer/vendor/phpseclib/phpseclib/phpseclib/Net/SFTP.php:392
PHP  16. phpseclib\Net\SSH2->_login() phar:///usr/local/bin/deployer/vendor/phpseclib/phpseclib/phpseclib/Net/SFTP.php:392
PHP  17. phpseclib\Net\SSH2->_connect() phar:///usr/local/bin/deployer/vendor/phpseclib/phpseclib/phpseclib/Net/SSH2.php:1879
PHP  18. user_error() phar:///usr/local/bin/deployer/vendor/phpseclib/phpseclib/phpseclib/Net/SSH2.php:1011


                                                  
  [RuntimeException]                              
  Unable to login with the provided credentials.  
                                                  


Exception trace:
 () at phar:///usr/local/bin/deployer/src/Server/Remote/PhpSecLib.php:89
 Deployer\Server\Remote\PhpSecLib->connect() at phar:///usr/local/bin/deployer/recipe/common.php:68
 {closure}() at n/a:n/a
 call_user_func() at phar:///usr/local/bin/deployer/src/Task/Task.php:73
 Deployer\Task\Task->run() at phar:///usr/local/bin/deployer/src/Executor/SeriesExecutor.php:39
 Deployer\Executor\SeriesExecutor->run() at phar:///usr/local/bin/deployer/src/Console/TaskCommand.php:82
 Deployer\Console\TaskCommand->execute() at phar:///usr/local/bin/deployer/vendor/symfony/console/Command/Command.php:259
 Symfony\Component\Console\Command\Command->run() at phar:///usr/local/bin/deployer/vendor/symfony/console/Application.php:878
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/local/bin/deployer/vendor/symfony/console/Application.php:195
 Symfony\Component\Console\Application->doRun() at phar:///usr/local/bin/deployer/vendor/symfony/console/Application.php:126
 Symfony\Component\Console\Application->run() at phar:///usr/local/bin/deployer/src/Deployer.php:99
 Deployer\Deployer->run() at phar:///usr/local/bin/deployer/bin/dep:62
 require() at /usr/local/bin/deployer:4


deploy [-p|--parallel]
```

I use deployer to deploy Symfony2 application

**Comment by elfet** [comment]
Yeah. Will change it shortly.

**Comment by elfet** [comment]
Fixed in v3.0.9. You can upgrade: `dep self-update`.
See #435 for more details.
